signal/src/voting_structs.rs

```diff
@@ -0,0 +1,23 @@
+use frame_support::pallet_prelude::{Encode, Decode};
```

Comment:
I'm not sure if we should keep structs and enums in separate places. For now, it's not critical, but let's discuss it later and use the same approach for all pallets.